DistFiles/xMatter/bloom-xmatter-mixins.jade

```diff
@@ -146,12 +146,8 @@ mixin factoryStandard-titlePage
 		#languageInformation
 			.languagesOfBook(data-book='languagesOfBook')
 			//- review: can we get rid of these "langName" classes?
```

Comment:
Agreed on the undesirability of that class, but also that now is not the time to make a bigger change than absolutely necessary.